admin_app/src/app/content/edit/page.tsx

```diff
@@ -224,7 +249,45 @@ const ContentBox = ({
             placeholder="Add Tags"
           />
         )}
+        filterOptions={(options, params) => {
+          const filtered = filter(options, params);
+          const { inputValue } = params;
+          const isExisting = options.some(
+            (option) => inputValue.toUpperCase() === option.tag_name
+          );
+
+          const isSelected = contentTags.some(
+            (tag) => inputValue.toUpperCase() === tag.tag_name
+          );
+
+          if (inputValue !== "" && !isExisting && !isSelected) {
+            filtered.push({ tag_id: 0, tag_name: `Add "${inputValue}"` });
+          }
+
+          return filtered;
+        }}
+        renderOption={(props, option) => {
+          const { key, ...newProps } =
+            props as React.HTMLAttributes<HTMLLIElement> & { key: React.Key };
+          if (option.tag_name) {
+            return (
+              <li key={key} {...newProps}>
+                <Button fullWidth onClick={() => handleNewTag(option.tag_name)}>
+                  {option.tag_name}
+                </Button>
+              </li>
+            );
+          }
+          return (
+            <li key={option.tag_id} {...newProps}>
+              {option.tag_name}
+            </li>
+          );
+        }}
         sx={{ width: "500px" }}
+        isOptionEqualToValue={(option, value) =>
```

Comment:
It is supposed to fix the tags filter warning. But apparently there is a bug on MUI side. They plan to fix it in the next version